### PR TITLE
Mavis

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92190)

### CODE REVIEW

**Comment by Trenly** [comment]
@msftbot zip

**Review by ItzLevvie — DISMISSED** [review]

**Comment by Trenly** [comment]
[Policy] Reset Feedback

**Review by stephengillie — DISMISSED** [review]

File: manifests/s/skymavis/mavishub/1.5.0/skymavis.mavishub.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: skymavis.mavishub
+PackageVersion: 1.5.0
+PackageLocale: en-US
+Publisher: Sky Mavis
+PackageName: Mavis Hub
```
Comment (stephengillie) [inline manifests/s/skymavis/mavishub/1.5.0/skymavis.mavishub.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.skymavis.com/
PackageName: Mavis Hub
PackageUrl: https://welcome.skymavis.com/download/
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**